src/v/raft/consensus.h

```diff
@@ -449,7 +453,7 @@ class consensus {
     };
 
     // precondition: is_elected_leader() must be true.
-    suppress_heartbeats_guard suppress_heartbeats(vnode);
+    track_inflight_appends_guard append_inflight(vnode);
```

Comment:
hah, what I meant was to add "track" to the function name, not the class name (so that function name has a verb in it) :)

Reply:
lmao 🤦 , let me fix it again.

Reply:
one more stamp please :-)